### DISCUSSION TITLE
App stopped working as of July 15th

### DISCUSSION THREAD

**Discussion by C3BT** [aidanmacgregor/Netflix_ATV_L3_DRM_Uncertified_Mod#43] [Q&A]
Anyone else forced to uninstall?

**Comment by Basdekruijf** [comment]
Aidan are you planning to update the app?

**Reply by lowelltortona** [reply]
Requesting if the app can be updated in order for it to run in an Android TV box.

**Reply by C3BT** [reply]
Did you try the 2 md apk? It worked for me on my Android TV

**Comment by C3BT** [comment]
I tried using the 2 md apk and it is working now fine

**Reply by Kay2045** [reply]
i cant find it, can you tell me the full name to search or a link?
